<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="windowFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>261</width>
      <height>461</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1111111</width>
      <height>1111111</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>98</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="stockStallionLogo">
        <property name="text">
         <string>Logo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stockStallionLogo_2">
        <property name="font">
         <font>
          <family>Century Gothic</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: Gainsboro;
border-style: solid;
border-width:2px;
border-radius:7px;
border-color: black;
font: 12pt &quot;Century Gothic&quot;;
</string>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>3</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: white;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameErrorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: transparent;</string>
     </property>
     <property name="text">
      <string>Put username constraints here</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailErrorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: transparent;</string>
     </property>
     <property name="text">
      <string>Please enter a valid E-mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordErrorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: transparent;</string>
     </property>
     <property name="text">
      <string>Put password constraints here</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordConfirmErrorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: transparent;</string>
     </property>
     <property name="text">
      <string>Passwords do not match</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>440</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Have an account?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>436</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 75 9pt &quot;Century Gothic&quot;;
text-decoration: underline;</string>
     </property>
     <property name="text">
      <string>Login Here</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
